linker/shared/src/main/scala/org/scalajs/linker/standard/StandardIRFileCache.scala

```diff
@@ -33,8 +33,6 @@ final class StandardIRFileCache(config: IRFileCacheConfig) extends IRFileCacheIm
    * that paying the cost for synchronization is lower than I/O.
    */
 
-  def this() = this(IRFileCacheConfig())
```

Comment:
This seems like a useful shorthand to keep around. Usually we offer a no-arg constructor as a forwarder for a config-base constructor with the default config. For example all JS envs have a constructor like that.

Reply:
Do you also want to not deprecate the convenience method in `StandardImpl` then?

Reply:
Yes, that would make sense.